.gitignore

```
@@ -10,6 +10,13 @@ packages/*/test/visual/screenshots/failed
 packages/*/test/visual/*/screenshots/*/failed
 packages/icons/test/visual/screenshots/failed
 
+packages/*/test/visual/screenshots/local-*
+packages/*/test/visual/*/screenshots/*/local-*
+packages/icons/test/visual/screenshots/local-*
+packages/*/test/visual/screenshots/local-*
+packages/*/test/visual/*/screenshots/*/local-*
+packages/icons/test/visual/screenshots/local-*
```

```suggestion
packages/*/test/visual/screenshots/local-*
packages/*/test/visual/*/screenshots/*/local-*
packages/icons/test/visual/screenshots/local-*
```
The last three lines are duplicates I believe. Also not sure why the icons has a separate rule, it looks like it would be covered by the first rule.